docs/01-app/01-getting-started/06-css.mdx

```diff
@@ -126,4 +180,113 @@ export default function RootLayout({ children }) {
 }
 ```
 
-External stylesheets must be directly imported from an npm package or downloaded and colocated with your codebase. You cannot use `<link rel="stylesheet" />`.
+> **Good to know:** External stylesheets must be directly imported from an npm package or downloaded and colocated with your codebase. You cannot use `<link rel="stylesheet" />`.
```

Comment:
It's worth testing this again, I think this should just work with React 19, but maybe still not something we want to recommend.

Reply:
Tested both. `<head><link rel=.../>` works. But `import` is still better because it combines and dedupes multiple imports of the same stylesheet, whereas link doesn't.